sentry_sdk/integrations/aiohttp.py

```diff
@@ -96,21 +98,18 @@ def setup_once():
 
         old_handle = Application._handle
 
+        @ensure_integration_enabled_async(AioHttpIntegration, old_handle)
         async def sentry_app_handle(self, request, *args, **kwargs):
             # type: (Any, Request, *Any, **Any) -> Any
-            hub = Hub.current
-            if hub.get_integration(AioHttpIntegration) is None:
-                return await old_handle(self, request, *args, **kwargs)
-
             weak_request = weakref.ref(request)
 
-            with Hub(hub) as hub:
-                with auto_session_tracking(hub, session_mode="request"):
+            with sentry_sdk.isolation_scope() as scope:
+                with auto_session_tracking_scope(scope, session_mode="request"):
                     # Scope data will not leak between requests because aiohttp
                     # create a task to wrap each request.
-                    with hub.configure_scope() as scope:
-                        scope.clear_breadcrumbs()
-                        scope.add_event_processor(_make_request_processor(weak_request))
+                    scope.generate_propagation_context()
```

Comment:
I know you mentioned this, but why is this necessary again?

Reply:
Not sure if it is, but it's a direct translation to what the `configure_scope` was doing in the background: https://github.com/getsentry/sentry-python/blob/6a2280eefc91933d2a46824fa1f7be8151c5d59c/sentry_sdk/hub.py#L597-L622

Reply:
Got it. Thanks!